modules/use-cases-architectures/pages/real-time-data-pipeline/02-create-decodable-objects.adoc

```diff
@@ -8,246 +8,198 @@ This guide is part of a series that creates a real-time data pipeline with {prod
 
 == The {product} connection info
 
-To connect {product} to Decodable, you need some information from your {product} tenant. In the {astra-ui-link}, navigate to the webstore-clicks streaming tenant.
+To connect {product} to Decodable, you need some information from your {product} tenant.
 
-image:decodable-data-pipeline/02/image4.png[]
+. In the {astra-ui-link} header, click icon:material-icons:apps[] **Applications**, select **Streaming**, and then click your `webstore-clicks` streaming tenant.

```

Comment:
```suggestion
. In the {astra-ui-link} header, click icon:material-icons:apps[] **Applications**, select **Streaming**, and then click your `webstore-clicks` streaming tenant.
+
```

Reply:
<img width="518" height="330" alt="Screenshot 2025-07-15 at 1 23 06 PM" src="https://github.com/user-attachments/assets/723c1e1c-e79d-4607-af79-c62adb4cd8f7" />
Pass thru to address Numbering